scheduler/src/cook/cors.clj

```diff
@@ -7,9 +7,11 @@
 (defn same-origin?
   "Returns true if the request is from the same origin as the provided origin header"
   [{:keys [headers scheme]}]
-  (let [{:strs [host origin]} headers]
+  (let [{:strs [host origin x-forwarded-proto]} headers
+        scheme (or x-forwarded-proto
```

Comment:
Should we name this something other than `scheme` to avoid confusion with the passed-in `scheme`?

Reply:
Will address.

Reply:
Done